javascript/builtins/TypedArray.json

```
@@ -1467,7 +1467,7 @@
                 "notes": "Was available in Aurora and Nightly channels only."
               },
               "firefox_android": {
-                "version_added": "38",
+                "version_added": "34",
```

This should be "16" like for Firefox Desktop.

---

You know you can make use of GitHub suggestions, right?
```suggestion
                "version_added": "16",
```

````markdown
```suggestion
                "version_added": "16",
```
````

---

I know that. Sorry if I don't always use it.

---

GitHub suggestions are a pretty new feature.  Not everyone's going to be using it.  Both methods are quite helpful.  🙂